components/DropdownMenu.tsx

```diff
@@ -23,33 +24,49 @@ type DropDownMenuProps = {
     | 'warning'
     | 'danger'
     | 'none'
+  bsPrefix?: string
```

Comment:
what is `bsPrefix`? Can you please give this a better name?

Reply:
I added comment describing how bsPrefix works in bootsrap.